lib/crypto/test/crypto_SUITE.erl

```diff
@@ -674,6 +674,7 @@ cipher_info(Config) when is_list(Config) ->
     #{type := _,key_length := _,iv_length := _,
         block_size := _,mode := _} = crypto:cipher_info(aes_128_cbc),
     {'EXIT',_} = (catch crypto:cipher_info(not_a_cipher)),
+    _ = [crypto:cipher_info(C) || C <- proplists:get_value(ciphers, crypto:supports())],
```

Comment:
These comprehensions should be `lists:foreach/2` calls, really -- and then you can delete the following line.

Reply:
Was a quick copy pasta from a larger snippet. I'll fix it, thanks.